action.yml

```
@@ -29,6 +33,8 @@ outputs:
     description: Changelog markdown for the minor release branch
   minor_version:
     description: Minor version string, e.g. "v1.39"
+  isValidChangelogEntry:
```

```suggestion
  is_valid_changelog_entry:
```

Let's make it snake_case as other parameters.